examples/morpheusvm/README.md

```diff
@@ -28,7 +28,7 @@ The first step to running this demo is to launch your own `morpheusvm` Subnet. Y
 can do so by running the following command from this location (may take a few
 minutes):
 ```bash
-./scripts/run.sh;
+killall avalanche-network-runner
```

Comment:
Replace single command script to make it more obvious what's happening

Reply:
Corrected this (swapped run and stop). Using killall here because that was the only command in `stop.sh`. I agree we should not use a blanket killall in general, but this is just getting rid of a one line script in favor of using the command directly.